Migrations/Code/LegacyClassesForIde.php

```diff
@@ -7,6 +7,11 @@
 }
 
 namespace {
+    /** @deprecated */
+    class Tx_Rnbase_Frontend_Plugin extends \TYPO3\CMS\Frontend\Plugin\AbstractPlugin
```

Comment:
You should not extend from base class. This is a deprecation notice for ide only.
```suggestion
    class Tx_Rnbase_Frontend_Plugin
```

Reply:
Please see my comment in https://github.com/digedag/rn_base/pull/246#discussion_r729648157 explaining why I think the `extends` is helpful.